### PR TITLE
Cockpit 287

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-287.html

### CODE REVIEW

File: _posts/2023-03-06-cockpit-287.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/287)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/285)
+* [cockpit-machines Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit-machines)
+* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit-machines)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/64)
+* [cockpit-podman Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit-podman)
+* [cockpit-podman Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit-podman)
```
Comment (skobyda) [inline _posts/2023-03-06-cockpit-287.md:108]: Bodhi is now ready:

```
* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/287)
* [Cockpit Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-6987aaacad)
* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-2e47853ed4)
* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/285)
* [cockpit-machines Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-99b4679f77)
* [cockpit-machines Fedora 37]https://bodhi.fedoraproject.org/updates/FEDORA-2023-9893be2f68()
* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/64)
* [cockpit-podman Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-82b8596f0f)
* [cockpit-podman Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-ff01f5ff9e)
```

Comment (garrett) [reply _posts/2023-03-06-cockpit-287.md:108]: But the release notes aren't? They're still preview mode. We shouldn't be editing them until they're regenerated in normal mode.

Comment (garrett) [reply _posts/2023-03-06-cockpit-287.md:108]: I can regenerate after lunch and replace the block manually though. Thanks!

**Comment by garrett** [comment]
What happened to the "Overview: Support for additional timeservers with chronyd" section? It's completely broken! :facepalm: 

Why wasn't I notified about this? I would've rejected it and required it to be fixed.

There's no PatternFly styling on the text entries and there's no spacing between elements. "Change" should be save. Arhghghghg!

File: _posts/2023-03-09-cockpit-287.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The data storage for "pinned" systemd units has changed. After upgrading to this Cockpit version, you need to re-pin units.
+
+## Overview: Support for additional timeservers with chronyd
+
+Cockpit can now configure additional time servers when chronyd is used to synchronize the clock. Previously, this was only supported with systemd-timesyncd.
+
+![screenshot of support for additional timeservers with chronyd]({{ site.baseurl }}/images/287-overview-support-for-additional-timeservers-with-chronyd.png)
+
+
```
Comment (garrett) [inline _posts/2023-03-09-cockpit-287.md:27]: We should remove the chronyd part. It was merged after the release. (And also needs a follow-up before next release: https://github.com/cockpit-project/cockpit/issues/18496)

```suggestion
```

File: _posts/2023-03-09-cockpit-287.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/287)
+* [Cockpit Fedora 38](https://bodhi.fedoraproject.org/updates/?releases=F38&packages=cockpit)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/64)
+* [cockpit-podman Fedora 38](https://bodhi.fedoraproject.org/updates/?releases=F38&packages=cockpit-podman)
+* [cockpit-podman Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit-podman)
```
Comment (garrett) [inline _posts/2023-03-09-cockpit-287.md:45]:
```suggestion
* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/287)
* [Cockpit Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-6987aaacad)
* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-2e47853ed4)
* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/285)
* [cockpit-machines Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-99b4679f77)
* [cockpit-machines Fedora 37]https://bodhi.fedoraproject.org/updates/FEDORA-2023-9893be2f68()
* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/64)
* [cockpit-podman Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-82b8596f0f)
* [cockpit-podman Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-ff01f5ff9e)
```

File: _posts/2023-03-09-cockpit-287.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+title: Cockpit 287
+author: garrett
+date: '2023-03-09'
+tags: cockpit, podman
+slug: cockpit-287
+category: release
+summary: ''
```
Comment (garrett) [inline _posts/2023-03-09-cockpit-287.md:8]:
```suggestion
summary: 'Pinned services update, esbuild developer information'
```

**Comment by garrett** [comment]
- [ ] We need to remove the image from this PR

What else needs to happen (aside from squashing) before this PR can be merged?

**Review by skobyda — APPROVED** [review]

**Comment by skobyda** [comment]
> * [x]  We need to remove the image from this PR

Done

Everything looks correct and great, let's get this merged

**Pull request merged**